src/Utils.js

```diff
@@ -236,15 +236,23 @@ TinCan client library
         /**
         @method parseURL
         @param {String} url
+        @param {Object} [options]
+            @param {Boolean} [options.allowRelative] Option to allow relative URLs
         @return {Object} Object of values
         @private
         */
-        parseURL: function (url) {
+        parseURL: function (url, options) {
```

Comment:
Other functions use 'cfg' for what seems to me to be a similar purpose to 'options' here. Is there a good reason not to use cfg here for consistency?